recipes/google-weather-tools/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "google-weather-tools" %}
+{% set version = "0.3.2" %}
+{% set git_rev = "conda-candidate-v0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/google/weather-tools.git
+  git_rev: "{{ git_rev }}"
+
+build:
+  skip: true  # [win]
```

Comment:
```suggestion
  noarch: python 
```

Reply:
@xhochy Thanks for your help and happy new year! Some of our runtime dependencies are not supported in Windows, what is the right way to release for Unix only if having **noarch** is mandatory?

Reply:
According to [this answer](https://github.com/conda-forge/staged-recipes/issues/11393#issuecomment-638423909), I assume the recipe should not be `noarch` since it's not supported in Windows. Please correct me if I'm wrong.

Reply:
No, if it's just that the dependencies are unavailable on Windows, that's fine for this to be noarch. It's only a problem with dependencies that are only install on Windows/not-Windows. noarch packages are generally only built on Linux systems, which will succeed. The missing dependencies will then keep it from being installable on Windows.

Reply:
I'm a bit confused by the discussion here. In my understanding, the solution to move forward is to configure this to be noarch. 

To add some general background context: We have not yet implemented windows support in the package we want to release (https://github.com/google/weather-tools/issues/64). If we set this to noarch, does that mean we have to fix this bug in order to proceed? 

Thanks for your inputs, @dopplershift and @xhochy.

Reply:
Well since you said:
> Some of our runtime dependencies are not supported in Windows

That's why I said noarch was the right answer. This is the first time it's been mentioned that it doesn't work properly on Windows. In that case, we can proceed with non-noarch.

Having said that though, it looks like the problem is just that you're using `scripts` for the executables. That's pretty easy to move to [`console_scripts`](https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html) which work on Windows and are supported by our noarch builds.